### PR TITLE
Fixes issue where fixed sidebar was being applied to item pages

### PR DESCRIPTION
**Pull request by edmadrid**
I didn't realize item pages were using same sidebar id (#sidebar) as static pages, so the item page sidebar was getting fixed in an undesirable way. 

Added a unique sidebar id for static pages.

### CODE REVIEW

**Comment by cgalarza** [comment]
I think the sidebar is still having an issue on the search results page. I think the selector needs to be tweaked.

**Comment by cgalarza** [comment]
`#static-text > #sidebar` might work

**Comment by cgalarza** [comment]
Yea, I was seeing it do that after i deployed this branch, but maybe i deployed the wrong one. It looks great now!

File: app/assets/stylesheets/partials/static-page.scss [hunk 1]
```diff
@@ -110,46 +115,35 @@
...
+    & .affix-top,
+    & .affix {
+      position: static;
+    }
 
-   #sidebar.affix {
-     position: fixed;
-     top: 0;
-   }
+    & #sidebar.affix-top {
```
Comment (cgalarza) [inline app/assets/stylesheets/partials/static-page.scss]: Do you still need to define `#sidebar.affix-top` or can it be `.affix-top`?

**Comment by edmadrid** [comment]
no need for #sidebar now, right. made the change.

**Comment by cgalarza** [comment]
Great, thanks! Did you deploy to dev?

**Comment by edmadrid** [comment]
ok deployed to dev.

**Pull request merged**